### ISSUE TITLE
name logger is not deffined

### ISSUE THREAD

**Issue by zagnox** [ShutdownRepo/pywhisker#26]
```
python3 pywhisker.py -d "domain.local" -u "user" -p "password" --target "target" --action "add"
[!] name 'logger' is not defined
```

I get this error whenever I try to run. I think it could be a prblem with the code. I installed all the requirements. Anyone else facing the same issue?

**Comment by sosohayb** [comment]
add this after inports guys  :

import logging

# Configure logging
logging.basicConfig(level=logging.INFO)
logger = logging.getLogger(__name__)

**Comment by lc8992** [comment]
This worked for me:

import logging 

logging.basicConfig(level=logging.INFO)
logger = logging.getLogger(__name__)

**Comment by TcchSquad** [comment]
Has anyone had any luck with this? when I try the suggestion about importing logging just creates a bunch of errors - anyone have any tricks or can share a working modified version?

**Comment by Dawn-McLean** [comment]
I've got the same issue.

**Comment by dankuers** [comment]
Quick fix that solved it for me after cloning and cd'ing into the main dir:
`git checkout ec30ba5759d57ead54341f58289090a9dc01249a`

**Comment by wacenw37** [comment]
There is a built-in Logger class in the script. Just add `logger = Logger()` to line 753 (Just below the class) and logging will work.

However it seems the current version of OpenSSL (I have 24.2.1) has issues with PKCS12. (I didn't spend a lot of time on this).

Running in a venv will work:
```
python3 -m venv venv
source venv/bin/activate
pip install cryptography==36.0.0 pyOpenSSL==22.0.0 impacket dsinternals rich
python3 pywhisker.py -d "domain.local" -u "user" -p "password" --target "target" --action "add"
```

**Comment by ShutdownRepo** [comment]
Should be fixed thanks to #24 
Please feel free to reopen if things don't work as they should 🙏
